README.md

```diff
@@ -1553,6 +1553,20 @@ A list of tool extensions for .NET Core Command Line (dotnet CLI), aka '.NET Cor
         <code>dotnet tool install -g <a href="https://www.nuget.org/packages/nyancat/">nyancat</a></code>
       </td>
     </tr>
+    <tr>
+      <td><code>OMNIA Low-code</code></td>
```

Comment:
```suggestion
      <td><code>omnia-cli</code></td>
```
OMNIA Low-code Is the name of your project. This should be the name of the command instead.